src/components/MatchPage/MatchPage.js

```diff
@@ -44,6 +46,8 @@ export function MatchPage() {
 
   return (
     <div className="matchPage">
+      <CirclesBackground />
```

Comment:
adding the TopNav and the new Figma designs have the circlesBackground in the MatchPage now